cellpack/bin/upload.py

```diff
@@ -8,6 +8,25 @@
 from cellpack.autopack.loaders.recipe_loader import RecipeLoader
 
 
+def get_recipe_metadata(loader):
+    """
+    Extracts and returns essential metadata from a recipe for uploading
+    """
+    try:
+        recipe_meta_data = {
+            "format_version": loader.recipe_data["format_version"],
+            "version": loader.recipe_data["version"],
+            "name": loader.recipe_data["name"],
+            "bounding_box": loader.recipe_data["bounding_box"],
+            "composition": {},
+        }
+        if "grid_file_path" in loader.recipe_data:
```

Comment:
Q: should we check for all keys in `loader.recipe_data` and add them to the `recipe_meta_data` (except `composition`)? This way we won't have to do a special check for `grid_file_path` or any new keys we add to the recipe down the line.

Reply:
Thank you for asking! I was thinking the same approach: iterating over recipe data and recording every key except for `objects`, `representations`, etc., to make the uploading more programmatic. However I ended up choosing to specify keys(for now) so we get to fully control which keys are displayed and needed in the metadata. Helping to prevent errors during data handling.

And you're right, if there are more new top-layer keys, we should think about a more dynamic way to manage it.